woodwork/tests/accessor/test_accessor_utils.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from woodwork.accessor_utils import init_series
+from woodwork.logical_types import Categorical, Datetime
+from woodwork.tests.testing_utils import xfail_dask_and_koalas
+
+
+def test_init_series_valid_conversion_specified_ltype(sample_series):
```

Comment:
Maybe also worth adding a check for specifying a logical type that's different from the one this series will get inferred as.

Reply:
Added!